src/main/java/menion/android/whereyougo/gui/activity/MainActivity.java

```diff
@@ -687,13 +675,6 @@ private void clearPackageFromMemory() {
         }).start();
     }
 
-    private void clickMap() {
-        MapDataProvider mdp = MapHelper.getMapDataProvider();
-        mdp.clear();
-        mdp.addCartridges(cartridgeFiles);
-        MainActivity.wui.showScreen(WUI.SCREEN_MAP, null);
-    }
-
```

Comment:
Please let this as own method. I would say that it is so better readable.

Reply:
I am sometimes amazed by what people find readable... Jumping 200 lines for four lines of code is an interesting definition of readable. But you are the boss...